app/Actions/Photo/Strategies/AddPhotoPartnerStrategy.php

```diff
@@ -60,6 +54,9 @@ public function do(): Photo
 		);
 		$videoStrategy->do();
 
+		// If the video has already been existing, we must copy over the checksum
```

Comment:
```suggestion
		// If the video is uploaded already, we must copy over the checksum
```

"has already been existing" just sounds awkward...